File: HiringAPI/Hiring/Domain/Dto/General/Auth/TokenDto.cs
using Domain.Entities.General;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Dto.General.Auth
{
    public class TokenDto
    {
        public string Token { get; set; }
        public DateTime Expiration { get; set; }
        public ApplicationUser CurrentUser { get; set; }
        public bool IsActive { get; set; }
    }

}

File: UI/src/app/models/auth/TokenDto.ts
export interface TokenDto {
  token: string;
  expiration: string;
  currentUser: ApplicationUserDto;
  isActive: boolean;
}
export interface ApplicationUserDto {
  userName: string | null;
  companyName: string | null;
  fullName: string | null;
  profilePicture: string | null;
}
